bcc/perf.go

```diff
@@ -180,3 +180,25 @@ func (pm *PerfMap) poll(timeout int) {
 		}
 	}
 }
+
+//export callback_to_lost
+func callback_to_lost(lost C.uint64_t) {
```

Comment:
Does this work? IIRC `callback_to_lost` gets passed the number of lost events, not a cookie value that could be used to lookup `callbackData`. Also, lost events should be send over a separate channel (in the elf package we accept a `lostChan`).

Reply:
@schu Yeah, you're right. I made it use `lostChan`, and updated relevant examples too. Force pushed.

Reply:
Adding a `lostChan` alone unfortunately isn't sufficient and `callbackData := lookupCallback(uint64(lost))` won't work, as we don't have the [`callbackDataIndex`](https://github.com/kinvolk/gobpf/blob/161be1f8caaddc56bd6577187f867c5057c327f9/bcc/perf.go#L117) in `callback_to_lost`, i.e. we need to find a different way to expose this to the user.

I would be ok with setting the callback to `nil` in this pull request and open a follow-up issues to address that separately.